src/contractTest/java/uk/gov/hmcts/dm/controller/provider/StoredDocumentSearchControllerProviderTest.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.dm.controller.provider;
+
+import au.com.dius.pact.provider.junitsupport.Provider;
+import au.com.dius.pact.provider.junitsupport.State;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import uk.gov.hmcts.dm.domain.StoredDocument;
+import uk.gov.hmcts.dm.security.Classifications;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+@Provider("dm_store_search_document _provider")
```

Comment:
Provider has extra spacer before underscore " _provider". Won't match consumer and will not verify.